src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs

```diff
@@ -17,11 +16,13 @@ public static partial class RuntimeInformation
         private const string FrameworkName = ".NET Core";
 #endif
 
+        private static string s_frameworkDescription;
+
         public static string FrameworkDescription
         {
             get
             {
-                return string.Format("{0} {1}", FrameworkName, typeof(object).GetTypeInfo().Assembly.GetName().Version);
+                return s_frameworkDescription ?? (s_frameworkDescription = $"{FrameworkName} {typeof(object).GetTypeInfo().Assembly.GetName().Version}");
```

Comment:
It's C# 6, interpolated strings:
https://msdn.microsoft.com/en-us/library/dn961160.aspx